### PR TITLE
Provisioning stack automation

### PR DESCRIPTION
**Pull request by sshveta**

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 6542476623180547571207aee03a65d0ecf00f27:
:hurtrealbad: There were some flake issues that need to be resolved in order to merge the pull request:
```
./cfme/services/catalogs/orchestration_template.py:31:101: E501 line too long (119 > 100 characters)
```
:trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 055ca22023cb6c2616cddb9202f00125dc8f8f61:
:hurtrealbad: There were some flake issues that need to be resolved in order to merge the pull request:
```
./cfme/services/catalogs/orchestration_template.py:31:101: E501 line too long (119 > 100 characters)
```
:trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 5c49cee4716cf132c5f25493904028c17d819dfd:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit abeef748ecf881fbed26a04858f1cec115c2e8e3:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit e19e7040a8d77faabe7981e41ea91dbb28a0a9fa:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by sshveta** [comment]
Blocked by https://bugzilla.redhat.com/show_bug.cgi?id=1220882

File: cfme/tests/services/test_provision_stack.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    orch_dialog.create_service_dialog(dialog)
+    yield dialog
+
+
+@pytest.yield_fixture(scope="function")
+def catalog():
+    cat_name = "cat_" + generate_random_string()
+    catalog = Catalog(name=cat_name, description="my catalog")
+    catalog.create()
+    yield catalog
```
Comment (mfalesni) [inline cfme/tests/services/test_provision_stack.py]: What is the purpose of a yield fixture when the yield is the final statement in the function?

Comment (seandst) [inline cfme/tests/services/test_provision_stack.py]: Good question. It seems like this and the dialog fixture should probably have some cleanup code after the yield

**Comment by dajoRH** [comment]
Lint report for commit f312a18155c69c0059b8fd0a42148bce0229a17b:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by seandst** [comment]
I tried to add the needs-rebase label, but failed. :)

Could you please squash this down to one commit?

**Comment by dajoRH** [comment]
Lint report for commit 839ba8ec6b70be0d9c86ee6db7184d1cb3d9057c:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit a61a7aa4d72f27d5d665b69d4005191fe351933b:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit db0252237785f057bc547f46fa2ea0737c769d71:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 68a30023d75b5da0ca18fadfd642b9829f8cfaf1:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit d20bc7662ac27d8026e97bd88fd94ee642f5618e:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 4539d38d28fd6260eb3d8ef9e1fbed84b5dc00ff:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 64c26f8ada8d6a019f1bd5f2ba6d4794af92d713:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Pull request merged**